### ISSUE TITLE
Define community constituency

### ISSUE THREAD

**Issue by fricklerhandwerk** [NixOS/foundation#71]
If the foundation is to be representative of something at all, this something needs to be defined.

1. What constitutes a community member?
2. How to get in?
3. How to get out?

This may be as simple as:
1. The person is listed somewhere
2. A PR to that list is accepted by someone who is entitled.
3. Renewal times out (you get removed from the list automatically) or you leave voluntarily.

**Comment by thufschmitt** [comment]
Dupe of #62, closing to keep things tidy

**Comment by fricklerhandwerk** [comment]
No, this is not about the foundation board, this is about the community it's supposed to represent.

**Comment by KiaraGrouwstra** [comment]
> We want constituents to have a vested interest in the project's success.

The premise here seems the project's impact is limited to its users.
This seems to presume technology has no externalities on those not directly opting to use it, which might simplify the reality.
Now that military entities use NixOS, non-consensual externalities seem barely hypothetical.

As such, I would wonder how we would define project success, and why such measures of success should necessarily be considered both self-evident and uncontroversial.

**Comment by fricklerhandwerk** [comment]
This was addressed by the constitutional assembly specifying eligibility to vote for steering committee members: https://github.com/NixOS/org/blob/main/doc/constitution.md

**Comment by infinisil** [comment]
As mentioned above, this is now addressed by having a community-representative SC in charge of community matters :)
